vimiv/gui/thumbnail.py

```diff
@@ -238,21 +313,25 @@ def open_selected(self):
         api.signals.load_images.emit([self.current()])
         api.modes.IMAGE.enter()
 
-    @api.keybindings.register("<ctrl>b", "scroll page-up", mode=api.modes.THUMBNAIL)
-    @api.keybindings.register("<ctrl>f", "scroll page-down", mode=api.modes.THUMBNAIL)
+    @api.keybindings.register(
+        ("<ctrl>b", "<page-up>"), "scroll page-up", mode=api.modes.THUMBNAIL
```

Comment:
While I don't see any harm in these keybindings, they should go into a new PR in case you want them merged upstream :blush: If merged upstream, they should probably also be included in the library for consistency.

Reply:
The reason I made those keybindings is that scrolling with `page-up`, `page-down`, arrow keys, and so on would not trigger the scroll function, and thus the icons would not be updated to the new position. I wonder if this scrolling behavior is a byproduct of the OS' standard window controls. Could maybe be overridden somewhere, though?

Reply:
I see, those are most likely default bindings of the `QListWidget` then. If we don't recognize a binding, we try to pass it to Qt. Makes sense to override in this case :+1: